### PR TITLE
Get artifacts from Saucelabs run

### PR DESCRIPTION
**Pull request by BrennanConroy**
I hope this is right

### CODE REVIEW

**Comment by natemcmaster** [comment]
We're adding required PR checks to ensure the repo compiles before you merge. Can you update this PR by rebasing on master?

```
git fetch origin master:master
git rebase master
```

Alternative: if you plan to 'squash merge' this PR, you can also use a merge to update this PR. This can be easier to manage if you have lots of changes in this PR.
```
git fetch origin master:master
git merge master
```

File: .azure/pipelines/signalr-daily-tests.yml [hunk 1]
```diff
@@ -4,6 +4,10 @@
 # Daily Tests for ASP.NET Core SignalR
 # These use Sauce Labs resources, hence they run daily rather than per-commit.
 
+artifacts:
```
Comment (natemcmaster) [inline .azure/pipelines/signalr-daily-tests.yml]: This isn't in the right location. This needs to be under "jobs > parameters"

```diff
   parameters:
     buildDirectory: src/SignalR
     buildArgs: "/p:DailyTests=true /p:SauceUser='$(asplab-sauce-labs-username)' /p:SauceKey='$(asplab-sauce-labs-access-key)'"
     agentOs: Windows
     jobName: SignalRDailyTests
     jobDisplayName: "SignalR Daily Tests"
+    artifacts:
+      publish: true
+      path: 'src/SignalR/artifacts/'
```

Comment (BrennanConroy) [reply .azure/pipelines/signalr-daily-tests.yml]: Yeah, I changed it on my internal testing branch and it ran but didn't look at that folder for artifacts

**Comment by BrennanConroy** [comment]
@natemcmaster Looks like there was a bug in our artifacts check, fixed it and tested for the private build and it works.

**Review by natemcmaster — APPROVED** [review]

**Pull request merged**